packages/utility/agent.ts

```diff
@@ -118,7 +118,14 @@ const Opportunities = {
   list: () => requests.get<Opportunity[]>("opportunities"),
 };
 
-const Ideas = {};
+const Ideas = {
+  get: async (params?: URLSearchParams) => 
+    requests.get<Idea[]>("idea-cards", params),
+  getIdea: async (id: string, params?: URLSearchParams) => 
+    requests.get<Idea>(`/idea-cards/${id}`, params),
+  post: async (url: string, body: {}) =>
+    requests.post<Idea>(url, body)
```

Comment:
I don't think having these as async functions make sense, they're going to be awaited in the app level.

Reply:
Okay. I have removed the async identifier here and tested that things are still working.